elaston/units.py

```diff
@@ -217,3 +225,11 @@ def __class_getitem__(cls, metadata):
 class Int:
     def __class_getitem__(cls, metadata):
         return Annotated[int, metadata]
+
+
+class OntoType(ABC):
+    pass
+
+
+def u(type_, /, units: str | None = None, otype: OntoType | None = None):
```

Comment:
I guess here we need to wait and see about use cases and a potential dataclass for allowable fields. We could go super general `def u(type_, /, **kwargs)` and just annotate use _whatever_ the user provides. The opposite end is to already define a `@dataclass(class Pyironnotations)` to hold these things and match the signature here with fields there. TBH beyond the units and a vague sense that ontologies are important to us, I don't know what we'd put in it or how we should name/type things.